### ISSUE TITLE
Bot is labeling issues made by org members, even if they are public

### ISSUE THREAD

**Issue by ethanhs** [getsentry/team-ospo#31]
e.g https://github.com/getsentry/sentry/issues/36587

**Comment by chadwhitacre** [comment]
Is this related to/fixed by https://github.com/getsentry/eng-pipes/pull/307?

**Comment by ethanhs** [comment]
Yes! I think you asked me to make this when we started talking about the issue. Closing :)

**Comment by ethanhs** [comment]
This seems to be happening still sometimes :/

e.g. https://github.com/getsentry/sentry-docs/issues/5522

**Comment by ethanhs** [comment]
This seems to be happening when the bot is labeling things to route to teams, it seems to not check whether the OP is a Sentry member.

**Comment by chadwhitacre** [comment]
Closing, hopefully done with https://github.com/getsentry/eng-pipes/pull/392.
